src/plugins/scroll-snap.js

```diff
@@ -0,0 +1,13 @@
+import prefix from '../prefix'
+
+export default {
+  supportedProperty: (prop) => {
+    if (prop.substring(0, 11) === 'scroll-snap') {
+      if (prefix.js === 'ms') {
+        return `${prefix.css}${prop}`
+      }
+      return prop
```

Comment:
You mean this line? Isn't it by default?

Reply:
For Safari 11, it shows errors for properties starts with 'scroll-snap'. And if no prefix for IE - it also shows error. So, I should check if it's IE.
Yes, it not return this props by default in browser. That's why I should do this plugin.